src/Microsoft.Azure.SignalR/Internals/ServiceUserIdFeature.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class ServiceUserIdFeature : IServiceUserIdFeature
```

Comment:
no need to have an interface? The class is enough?

Reply:
Make senses. Fixed.